config/default.yml

```diff
@@ -4,6 +4,10 @@ AllCops:
     Patterns:
     - _spec.rb
     - "(?:^|/)spec/"
+  FactoryGirl:
```

Comment:
IMO it should still be prefixed `RSpec/` e.g. having nested namespace here. Also it should have an option to disable the whole namespace, similar to rubocop's railscops